gobblin-core/src/main/java/gobblin/source/extractor/extract/kafka/KafkaExtractor.java

```diff
@@ -41,6 +46,9 @@
 
   protected Iterator<MessageAndOffset> messageIterator;
   protected long nextWatermark;
+  private long totalRecordSize;
```

Comment:
Is it possible to use `gobblin-metrics`, or this is what @ibuenros will be working on?